packages/core/src/components/fieldset/fieldset.css

```diff
@@ -0,0 +1,28 @@
+.hds-fieldset {
+  --border-style: 0 none;
+  --padding-top: 4px;
+  --padding-sides: 0;
+  --padding-bottom: var(--spacing-s);
+
+  border: var(--border-style);
+  box-sizing: border-box;
+  padding: var(--padding-top) var(--padding-sides) var(--padding-bottom)
+}
+
+.hds-fieldset-legend {
+  font-weight: bold;
+}
+
+.hds-fieldset--border {
+  --border-style: 2px inset var(--color-black-90);
+  --padding-top: 2px;
+  --padding-sides: 12px;
+  --padding-bottom: var(--spacing-s);
```

Comment:
Does this override the padding-bottom variable defined in line 5? It seems to have the same value, so not sure if it is necessary?

Reply:
I will remove this